### PR TITLE
#52 add intro for fyde app

### PR DESCRIPTION
**Pull request by luisalima**
This adds the intro for the Fyde app section. It still needs a lot more detail as outlined in #52 , but it's a start. :)

### CODE REVIEW

File: fyde-app/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+has_children: false
+nav_order: 3
+permalink: /fyde-app
+---
+# Fyde App
+{: .no_toc }
+
+The Fyde app bundles the end user UI and the Fyde agent. It is available on the following platforms:
+
+* iOS - available on the [App Store](https://www.fyde.com/downloads)
```
Comment (oNaiPs) [inline fyde-app/README.md]: @luisalima there are platform specific links.

https://www.fyde.com/downloads/ios
https://www.fyde.com/downloads/linux
https://www.fyde.com/downloads/macos
https://www.fyde.com/downloads/windows

You're also missing Android
https://www.fyde.com/downloads/android

File: fyde-app/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* MacOS - available on the [App Store](https://www.fyde.com/downloads)
+* Windows - available on the [Microsoft Store](https://www.fyde.com/downloads)
+* Linux - repos soon available for Debian and Arch.
+
+## Fyde Agent
```
Comment (oNaiPs) [inline fyde-app/README.md]: @luisalima perhaps you can update the glossary, namely https://fyde.github.io/docs/glossary/#fyde-app to be in sync with here

File: fyde-app/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+The Fyde agent intersects requests at the network layer. When a DNS request matches a protected resource, it injects a reply, pointing the domain to an internally accessible marker IP address that represents the resouce. It then, in parallel, initiates a request to the Access Policy Engine to grant permission to the resouce, and establishes an mTLS connection to the Access Proxy that is associated with the resource. It also includes a DNS security engine that blocks requests that match blacklists configured at the Enterprise Console level.
```
Comment (rasmartins) [inline fyde-app/README.md]: "resource" instead of "resouce"

File: fyde-app/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Fyde App
+has_children: false
+nav_order: 3
```
[the remark this answers is not in the document]
Comment (GMartinez-Sisti) [reply fyde-app/README.md]: Done.

File: fyde-app/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (GMartinez-Sisti) [reply fyde-app/README.md]: Done

Comment (GMartinez-Sisti) [reply fyde-app/README.md]: Updated

**Comment by GMartinez-Sisti** [comment]
Just updated with the suggestions. Also removed descriptions from the download links, it looked misleading saying that they were available in X store but the link redirects to fyde.com.

File: fyde-app/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+* [iOS](https://www.fyde.com/downloads/ios){:target="_blank"}
+* [Android](https://www.fyde.com/downloads/android){:target="_blank"}
+* [MacOS](https://www.fyde.com/downloads/macos){:target="_blank"}
+* [Windows](https://www.fyde.com/downloads/windows){:target="_blank"}
+* [Linux](https://www.fyde.com/downloads/linux){:target="_blank"}
+
+## Fyde Agent
+
+The Fyde agent intersects requests at the network layer. When a DNS request matches a protected resource, it injects a reply, pointing the domain to an internally accessible marker IP address that represents the resource. It then, in parallel, initiates a request to the Access Policy Engine to grant permission to the resource, and establishes an mTLS connection to the Access Proxy that is associated with the resource.
```
Comment (rasmartins) [inline fyde-app/README.md]: Maybe intercepts instead of intersects ?

Comment (GMartinez-Sisti) [reply fyde-app/README.md]: Nice catch.

**Pull request merged**